lib/register.php

```diff
@@ -451,6 +451,66 @@ function gutenberg_register_rest_api_post_revisions() {
 }
 add_action( 'rest_api_init', 'gutenberg_register_rest_api_post_revisions' );
 
+/**
+ * Includes the value for the custom field `post_type_capabities` inside the REST API response of user.
+ *
+ * TODO: This is a temporary solution. Next step would be to edit theWP_REST_Users_Controller,
```

Comment:
Missing space at `theWP_REST_Users_Controller`.

Wondering if we should be grouping these things that are expected to be fixed directly in core. Until then, does this function, as well as the next, make more sense in `lib/compat.php`? It might make them more visible.